R/plotColData.R

```diff
@@ -2,35 +2,71 @@
 #' @param x String specifying the column-level metadata to show on the x-axis.
-#' Alternatively, an \link{AsIs} vector or data.frame, see \code{?\link{retrieveCellInfo}}.
-#' If \code{NULL}, nothing is shown on the x-axis.
-#' @param colour_by Specification of a column metadata field or a feature to colour by, see the \code{by} argument in \code{?\link{retrieveCellInfo}} for possible values. 
-#' @param shape_by Specification of a column metadata field or a feature to shape by, see the \code{by} argument in \code{?\link{retrieveCellInfo}} for possible values. 
-#' @param size_by Specification of a column metadata field or a feature to size by, see the \code{by} argument in \code{?\link{retrieveCellInfo}} for possible values. 
-#' @param order_by Specification of a column metadata field or a feature to order points by, see the \code{by} argument in \code{?\link{retrieveCellInfo}} for possible values. 
-#' @param by_assay_name A string or integer scalar specifying which assay to obtain expression values from, 
-#' for use in point aesthetics - see \code{?\link{retrieveCellInfo}} for
-#' details (also alias \code{by_exprs_values} is accepted for this argument).
+#'   Alternatively, an \link{AsIs} vector or data.frame, see
+#'   \code{?\link{retrieveCellInfo}}. If \code{NULL}, nothing is shown on the
+#'   x-axis.
+#' @param colour_by Specification of a column metadata field or a feature to
+#'   colour by, see the \code{by} argument in \code{?\link{retrieveCellInfo}}
+#'   for possible values.
+#' @param shape_by Specification of a column metadata field or a feature to
+#'   shape by, see the \code{by} argument in \code{?\link{retrieveCellInfo}} for
+#'   possible values.
+#' @param size_by Specification of a column metadata field or a feature to size
+#'   by, see the \code{by} argument in \code{?\link{retrieveCellInfo}} for
+#'   possible values.
+#' @param order_by Specification of a column metadata field or a feature to
+#'   order points by, see the \code{by} argument in
+#'   \code{?\link{retrieveCellInfo}} for possible values.
+#' @param by_assay_name A string or integer scalar specifying which assay to
+#'   obtain expression values from, for use in point aesthetics - see
+#'   \code{?\link{retrieveCellInfo}} for details (also alias
+#'   \code{by_exprs_values} is accepted for this argument).
 #' @param by_exprs_values Alias for \code{by_assay_name}.
-#' @param other_fields Additional cell-based fields to include in the data.frame, see \code{?"\link{scater-plot-args}"} for details.
+#' @param other_fields Additional cell-based fields to include in the
+#'   data.frame, see \code{?"\link{scater-plot-args}"} for details.
 #' @param swap_rownames Column name of \code{rowData(object)} to be used to
-#'  identify features instead of \code{rownames(object)} when labelling plot
-#'  elements.
+#'   identify features instead of \code{rownames(object)} when labelling plot
+#'   elements.
 #' @param color_by Alias to \code{colour_by}.
```

Comment:
Can you remove the unrelated doc changes? Thanks

Reply:
I did and resolved the conflicts